sherpa/bin/pruned_stateless_emformer_rnnt2/streaming_client.py

```diff
@@ -57,6 +58,7 @@ def get_args():
     parser.add_argument(
         "sound_file",
         type=str,
+        required=True,
```

Comment:
This one causes CI errors:
https://github.com/k2-fsa/sherpa/runs/7459677322?check_suite_focus=true

```
  File "sherpa/bin/conformer_rnnt/offline_client.py", line 99, in main
    args = get_args()
  File "sherpa/bin/conformer_rnnt/offline_client.py", line 63, in get_args
    help="The input sound file(s) to transcribe. "
  File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/argparse.py", line 1341, in add_argument
    kwargs = self._get_positional_kwargs(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/argparse.py", line 1453, in _get_positional_kwargs
    raise TypeError(msg)
TypeError: 'required' is an invalid argument for positionals
```

Reply:
This is what the CI complains about.

Reply:
it's a different file and this argument is used inconsistently, I think.I can add  `nargs='+'` and remove the `required=True` but if you want but I think this works. Could you show me the ci output for this file?
y.

Reply:
Oh, sorry. I made a mistake. The CI is for a different file.

Reply:
I can still homogenize the clients to make them all support multiple files, if you want

Reply:
> I can still homogenize the clients to make them all support multiple files, if you want

Thanks. That is a good idea.